setup.py

```diff
@@ -62,213 +70,154 @@ def get_ext_filename_without_platform_suffix(filename):
+with open(os.path.join(PYTHON_PACKAGE_DIR, "README.rst"), encoding="utf-8") as f:
     long_description = f.read()
 
 # Get the current version for the python package from the configure.ac file
-config_path = os.path.join(here, 'version.txt')
-with open(config_path, encoding='utf-8') as f:
+config_path = os.path.join(REPO_ROOT_DIR, "version.txt")
+with open(config_path, encoding="utf-8") as f:
     version = f.readline().strip()
 
 try:
-    current_git_hash = subprocess.check_output(['git', 'rev-parse', '--short', 'HEAD']).decode('ascii').strip()
+    current_git_hash = (
+        subprocess.check_output(["git", "rev-parse", "--short", "HEAD"])
+        .decode("ascii")
+        .strip()
+    )
     if current_git_hash:
         version = version + "+" + current_git_hash
-except FileNotFoundError as e:
+except FileNotFoundError:
     pass
-except subprocess.CalledProcessError as e:
+except subprocess.CalledProcessError:
     pass
 
 setup(
-    name='vowpalwabbit',
+    name="vowpalwabbit",
     version=version,
-    python_requires='>=3.6',
-    description='Vowpal Wabbit Python package',
+    python_requires=">=3.6",
+    description="Vowpal Wabbit Python package",
     long_description=long_description,
-    url='https://github.com/VowpalWabbit/vowpal_wabbit',
-    author='Scott Graham',
-    author_email='scott.d.graham@gmail.com',
-    license='BSD 3-Clause License',
+    url="https://github.com/VowpalWabbit/vowpal_wabbit",
+    author="Scott Graham",
+    author_email="scott.d.graham@gmail.com",
+    license="BSD 3-Clause License",
     classifiers=[
-        'Development Status :: 4 - Beta',
-        'Intended Audience :: Science/Research',
-        'Topic :: Scientific/Engineering',
-        'Topic :: Scientific/Engineering :: Information Analysis',
-        'License :: OSI Approved :: BSD License',
-        'Programming Language :: Python :: 3',
+        "Development Status :: 4 - Beta",
+        "Intended Audience :: Science/Research",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Scientific/Engineering :: Information Analysis",
+        "License :: OSI Approved :: BSD License",
+        "Programming Language :: Python :: 3",
     ],
-    keywords='fast machine learning online classification regression',
-    package_dir={'' : os.path.relpath(pkg_path)},
-    packages=find_packages(where=pkg_path),
-    platforms='any',
+    keywords="fast machine learning online classification regression",
+    package_dir={"": os.path.relpath(PYTHON_PACKAGE_DIR)},
+    packages=find_packages(where=PYTHON_PACKAGE_DIR),
     zip_safe=False,
     include_package_data=True,
-    ext_modules=[CMakeExtension('pylibvw')],
-    distclass=Distribution,
-    cmdclass={
-        'build_ext': BuildPyLibVWBindingsModule,
-        'clean': Clean,
-        'sdist': Sdist,
-        'test': Tox,
-        'install_lib': InstallLib
-    },
-    # tox.ini handles additional test dependencies
-    tests_require=['tox'],
```

Comment:
what's the deal with the checked in file tox.ini?

Reply:
Tox from what I can tell is a testing framework for testing different python versions. We get this testing coverage through the CI matrix and tox hasn't been used for a long time. I think it is safe to remove